PokemonCard-v1.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+
+function PokemonCard() {
+
+ const [pokemonData, setPokemonData ] = useState({})
```

Comment:
Als je een makkelijk onderscheid wil hebben tussen of het al wél of niet is ingeladen, zou ik hier voor default waarde van `undefined` gaan
```suggestion
 const [pokemonData, setPokemonData ] = useState();
```

Reply:
Excuses Zwen,

Ik zie dat dit een oude versie is die ik als los .js bestand er nog in had laten staan. Ik zet deze voor nu op Resolve